demoScripts/quickstart-nodejs.md

```diff
@@ -0,0 +1,529 @@
+
+1. In the **App Service** explorer in Visual Studio code, expand the node for the new app, right-click **Application Settings**, and select **Add New Setting**:
+
+    ![Add app setting command](media/quickstart-nodejs/add-setting.png)
+
+1. Enter `SCM_DO_BUILD_DURING_DEPLOYMENT` for the setting key.
+1. Enter `true` for the setting value.
+
+    This app setting enables build automation at deploy time, which automatically detects the start script and generates the *web.config* with it.
+
+1. In the **App Service** explorer, select the **Deploy to Web App** icon again, confirm by clicking **Deploy** again.
+1. Wait for deployment to complete, then select **Browse Website** in the notification popup. The browser should display the Express default page.
+
+-----
+
+> [!div class="nextstepaction"]
+> [I ran into an issue](https://www.research.net/r/PWZWZ52?tutorial=node-deployment-azure-app-service&step=deploy-app)
+
+::: zone-end
+
+:::zone target="docs" pivot="development-environment-cli"
+
+In the terminal, ensure you're in the *myExpressApp* directory, and deploy the code in your local folder (*myExpressApp*) using the [az webapp up](/cli/azure/webapp#az-webapp-up) command:
+
+# Deploy to Linux
+
+```azurecli
+az webapp up --sku F1 --name $APP_NAME
+```
+* Add `--os-type Windows` to deploy to Windows:
+
+-----
+
+- If the `az` command isn't recognized, ensure you have the Azure CLI installed as described in [Set up your initial environment](#set-up-your-initial-environment).
+- The `--sku F1` argument creates the web app on the Free pricing tier, which incurs a no cost.
+- You can optionally include the argument `--location <location-name>` where `<location_name>` is an available Azure region. You can retrieve a list of allowable regions for your Azure account by running the [`az account list-locations`](/cli/azure/appservice#az-appservice-list-locations) command.
+- The command creates a Linux app for Node.js by default. To create a Windows app instead, use the `--os-type` argument. 
+- If you see the error, "Could not auto-detect the runtime stack of your app," ensure you're running the command in the *myExpressApp* directory (See [Troubleshooting auto-detect issues with az webapp up](https://github.com/Azure/app-service-linux-docs/blob/master/AzWebAppUP/runtime_detection.md)).
+
+The command may take a few minutes to complete. While running, it provides messages about creating the resource group, the App Service plan, and the app resource, configuring logging, and doing ZIP deployment. It then gives the message, "You can launch the app at http://&lt;app-name&gt;.azurewebsites.net", which is the app's URL on Azure.
+
+<pre>
```

Comment:
This looks like the results block for above, just not formatted as we need it.

Should we support <pre> tags as well as backticks for results blocks?